ProvisionSample/Program.cs

```diff
@@ -227,12 +227,16 @@ static void Main(string[] args)
 
                         if (string.IsNullOrWhiteSpace(workspaceId))
                         {
-                            Console.Write("Workspace ID:");
+                            Console.Write("Workspace Id:");
                             workspaceId = Console.ReadLine();
                             Console.WriteLine();
                         }
 
-                        await UpdateConnection(workspaceCollectionName, workspaceId);
+                        Console.Write("Dataset Id:");
+                        var datasetId = Console.ReadLine();
```

Comment:
please ensure this is  valid GUID. it is more robust and it will hint the users to use the guid and not the index